capstone/src/main/java/com/step902020/capstone/Event.java

```diff
@@ -31,8 +31,8 @@
   @Field(name="eventLongitude")
   double eventLongitude;
 
-  @Field(name="foodAvaliable")
-  Boolean foodAvaliable;
+  @Field(name="foodAvailable")
```

Comment:
Ah, both you and Olivia fixed it in your PRs. Might not even be a merge conflicts, since the result is the same.